scrapy/contrib/loader/processor.py

```diff
@@ -63,6 +63,32 @@ def __call__(self, values):
         return values
 
 
+class JsonProcessor(object):
+    """
+        Given a jmespath, will search for it in the set of scraped values
+        Requires : jmespath, ast modules
+    """
+    def __init__(self,jpath):
+        self.json_path = jpath
+        
+    def __call__(self,values):
+        jsonValues = []
+        try:
+            import jmespath,ast,json
+        except:
+            print "JsonProcessor module requires the jmespath library to function. No processing will happen in its abscence"
```

Comment:
When I said `required`, I meant for the Scrapy library. So it should raise the exception on the import of this Processor.